src/beautifiers/index.coffee

```diff
@@ -325,6 +325,12 @@ module.exports = class Beautifiers extends EventEmitter
         languages = @languages.getLanguages({grammar, extension: fileExtension})
         logger.verbose(languages, grammar, fileExtension)
 
+        # use fallback if configured
+        unsupportedLanguageFallback = atom.config.get("atom-beautify.unsupportedLanguageFallback")
+        if languages.length < 1 && unsupportedLanguageFallback.charAt(0) != '-'
+          languages = [require('../languages/' + unsupportedLanguageFallback)]
```

Comment:
@Glavin001 
I wanted to do that... but the problem is that I need an enum of all of the supported languages to populate the config dropdown (https://github.com/Glavin001/atom-beautify/blob/master/src/config.coffee), so I use `languageNames` from https://github.com/Glavin001/atom-beautify/blob/master/src/languages/index.coffee#L14

Then, for example, one of the languageNames that I might pick is `'javascript'`

`@languages.getLanguages` wants `name` to be something case sensitive like `'JavaScript'` which is not available in any one enum, but only from each individual language module
https://github.com/Glavin001/atom-beautify/blob/master/src/languages/javascript.coffee#L11

Do you have a suggestion for how to get the enum differently?